test/unit-test/ota_config.h

```diff
@@ -40,6 +40,9 @@
 /* Lower request momentum so that retry fails faster. */
 #define otaconfigMAX_NUM_REQUEST_MOMENTUM       3
 
+/* Make number of blocks per mqtt request larger so we can hit some branch. */
```

Comment:
nit: let's reword this for clarity, `/* Use larger number of blocks per mqtt request to increase branch coverage. */`